listenbrainz/influx-writer/influx-writer.py

```diff
@@ -42,6 +42,11 @@ def __init__(self):
         self.time = 0
 
 
+    def on_connection_closed(self, connection, reply_code, reply_text):
```

Comment:
there is no static method as a go between -- how does this work?

Reply:
this is how it was done in these [usage example docs](https://pika.readthedocs.io/en/0.10.0/examples/asynchronous_consumer_example.html)

but i've changed it to using a static method as done for the other callback.

Reply:
Wow, I had no idea that was possible -- when I tried it, I had no luck. But it seems so:

https://stackoverflow.com/questions/10003833/how-does-python-distinguish-callback-function-which-is-a-member-of-a-class

(I'll remove those static functions next time I'm mucking around there)